pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -202,7 +206,20 @@ def __call__(self, value, system):
             for element in text_element:
                 restriction_on_landownership[element] = '\n'.join(restriction_on_landownership[element])
 
-        extract_dict['RealEstate_RestrictionOnLandownership'] = list(theme_restriction.values())
+        restrictions = list(theme_restriction.values())
+        for restriction in restrictions:
+            legends = {}
+            for legend in restriction['Legend']:
+                type = legend['TypeCode']
+                if type in legends:
+                    for item in ['Area', 'Length', 'PartInPercent']:
+                        if item in legend:
+                            legends[type][item] += legend[item]
```

Comment:
I get an `KeyError: 'Length'` here because there are legen[item'] = None coming in.

Reply:
This mean that we can have tow element of the same `TypeCode`, with deferent geometry type?

Reply:
No, normally not. But I think there might be items where Area is set and Length defined as None? where the attribute Length should not exist

Reply:
ok ...

Reply:
Corrected, can you try again?